distribution/changelog.txt

```diff
@@ -18,6 +18,7 @@
 - Fix: [#7382] Opening the mini-map reverts the size of the land tool to 1x1, regardless of what was selected before.
 - Fix: [#7402] Edges of neigbouring footpaths stay connected after removing a path that's underneath a ride entrance.
 - Fix: [#7436] Only the first 32 vehicles of a train can be painted.
+- Fix: [#7424] Auto-rotate shops ignore queue lines
```

Comment:
We only put fixes in the changelog to bugs that were in the previous version. I think auto-rotate shop was merged after the release of 0.1.2.

Reply:
I 🗑 the line